x-pack/plugins/observability/public/pages/alert_details/mock/index.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TopAlert } from '../../alerts';
+
+export const tags: string[] = ['tag1', 'tag2', 'tag3'];
+
+export const alert: TopAlert | null = {
```

Comment:
nit: the type is not null

```suggestion
export const alert: TopAlert = {
```

Reply:
done 👍